src/main/java/net/snowflake/client/log/JDK14JCLWrapper.java

```diff
@@ -17,11 +17,11 @@ public JDK14JCLWrapper(String className) {
   }
 
   public void debug(Object msg) {
-    logger.debug(String.valueOf(msg));
```

Comment:
I removed all debug/warning/info messages from apache http logs, but error messages can still get through.

Reply:
hmm, In my understanding, this will remove all logs and not just for apache http messages? How do you distinguish between regular log and apache http log?

Reply:
We have 2 wrapper classes just for apache https logs(JDK14JCLWrapper.javaand SLF4JJCLWrapper.java) and 2 actual log classes (JDK14Logger.java and SLF4JLogger.java) where the actual JDBC driver logs pass through. I made modifications to the Wrapper classes only, which do not affect the JDBC driver logs. I tested to be sure.

Reply:
IMHO, Apache HttpClient package uses JCL (Jakarta Commons Logging), and this  JCL Wrapper class controls all logs redirected from apache httpclient package. In other words, this logger will be used by apache httpclient package for logging. So if we do nothing here, then no logs will be generated. @sfc-gh-mknister have you tested the result manually?

Reply:
@sfc-gh-mknister , ok, now I understand how this work. Can you take back WARNING and INFO? I don't believe we need DEBUG but we may want to have WARNING and INFO for any hint.

Reply:
Yes, I tested manually. Here are links to the files with and without these changes: [https://drive.google.com/file/d/1bEhuLDTzzOX0Fd9iDnUmNDcTdS2ZYqIa/view?usp=sharing](log with these changes)

and [https://drive.google.com/file/d/1x7n9fzQ_3-MKfi5nfQT_7jpz0NsV4FTM/view?usp=sharing](log without these changes)

Note that all log lines with "apache" are gone in the first log.

Reply:
@sfc-gh-stakeda sounds good, I added WARN and INFO back in